### PR TITLE
Qol 8423 search results   add filter results by to the search results page global search]

### PR DESCRIPTION
**Pull request by asifamingov**

### CODE REVIEW

**Review by duttonw — APPROVED** [review]
seems ok

**Comment by asifamingov** [comment]
> seems ok

Added Dist folder files to sync with Gitbridge. Committed some other files on top of the last commit.

File: global-search/src/app.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    /**
+     * trackRadioButtonChanges function track radio button changes when a user interacts with the form to set a session
+     * @return {undefined}
+     * */
+    trackRadioButtonChanges() {
+        var  selectedRadioBtnFromSession = sessionStorage.getItem('selectedRadiobutton');
+        $('input[type=radio][name=filterBy]').on('change', function() {
+            var selectedVal = $(this).val();
+            switch (selectedVal) {
+                case 'qld':
```
Comment (devonpis) [inline global-search/src/app.js:99]: can do :
```
case 'qld':
case 'custom':
 sessionStorage.setItem(`selectedRadiobutton`, selectedVal);
break;
```

File: global-search/src/app.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                    sessionStorage.setItem(`selectedRadiobutton`, selectedVal);
```
Comment (devonpis) [inline global-search/src/app.js:100]: for consistency could use `fc` as prefix for the key
ie. `fcSelectedRadiobutton `

File: global-search/src/styles.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#qg-two-col-aside.wide #qg-primary-content {
```
Comment (devonpis) [inline global-search/src/styles.css:1]: is this style something we could move to swe in the future?

**Review by devonpis — APPROVED** [review]

File: global-search/src/app.js (hunk 1 above)
Comment (asifamingov) [reply global-search/src/app.js]: updated in the recent commit

File: global-search/src/app.js (hunk 2 above)
Comment (asifamingov) [reply global-search/src/app.js]: updated in the recent commit

**Pull request merged**